README.md

```diff
@@ -393,6 +395,31 @@ with options as a JSON string of the plugin array:
 "importOrderParserPlugins": []
 ```
 
+#### `importOrderCaseSensitive`
+
+**type**: `boolean`
+
+**default value**: `false`
+
+A boolean value to enable case-sensitivity in the sorting algorithm
+used to order imports within each match group.
+
+For example, when false (or not specified):
+
+```javascript
+import ExampleComponent from './ExampleComponent';
+import ExamplesList from './ExamplesList';
+import ExampleWidget from './ExampleWidget';
+```
+
+compared with `"importOrderCaseSensitive": true`:
+
+```javascript
+import ExampleComponent from './ExampleComponent';
+import ExampleWidget from './ExampleWidget';
+import ExamplesList from './ExamplesList';
+```
+
```

Comment:
Since this will also sort ImportSpecifiers (`import {cat, Dog} from "C";` -> `import {Dog, cat} from "C";`), I think we should probably also include that in the readme here.